frontend/src/App.tsx

```diff
@@ -104,6 +105,7 @@ const App = (): React.ReactElement => {
                 value={dispatchPostingContextUpdate}
               >
                 <Router>
+                  {authenticatedUser ? "" : <AuthNavbar />}
```

Comment:
```suggestion
                  {authenticatedUser ? undefined : <AuthNavbar />}
```
would this make more sense semantically?